hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -310,6 +310,14 @@ public class HoodieClusteringConfig extends HoodieConfig {
           + "Please exercise caution while setting this config, especially when clustering is done very frequently. This could lead to race condition in "
           + "rare scenarios, for example, when the clustering completes after instants are fetched but before rollback completed.");
 
+  public static final ConfigProperty<String> ROLLBACK_PENDING_CLUSTERING_WHEN_DISABLED = ConfigProperty
+      .key("hoodie.rollback.pending.clustering.when.disabled")
```

Comment:
Could we replace the key `hoodie.rollback.pending.clustering.when.disabled` with `hoodie.clustering.rollback.pending.replacecommit.when.disabled`? The naming of `hoodie.rollback.pending.clustering.when.disabled` is confusing for users whether to disable clustering or rollback.